src/app/shared/interfaces/records-to-compare.ts

```diff
@@ -0,0 +1,5 @@
+import { QueryResult, RecordsPreview } from '../interfaces';
+export interface RecordsToCompare {
```

Comment:
what is this?

Reply:
Its an interface for the Preview function that fetches the old and new records to see the changes.